### ISSUE TITLE
Update Operator to install next release of Tekton 

### ISSUE THREAD

**Issue by anneqm** [kabanero-io/kabanero-operator#329]
Upgrade operator to install Tekton Triggers 0.1.0 and Tekton Dashboard with Webhooks Extension 0.3.0.  Prerequisites for Knative and Istio are no longer needed for Tekton (but may be needed for other upstream projects).

**Comment by mnuttall** [comment]
Note that Tekton Triggers now ship with the Openshift Pipelines Operator v0.8, available in the canary channel as of November 27th.

**Comment by anneqm** [comment]
Please note that v0.3.0 releases of the Tekton Dashboard and Webooks Extension is now available at https://github.com/tektoncd/dashboard/releases/tag/v0.3.0.

**Comment by kaczyns** [comment]
Our work to-date on this is being done here:
https://github.com/kabanero-io/kabanero-operator/tree/tekton-triggers

We are currently using openshift-pipelines operator v0.8.1, and tekton dashboard v0.3.0.  The pipelines team is using this branch for their testing.

**Comment by marikaj123** [comment]
@kaczyns - Is this issue ready to close?

**Comment by kaczyns** [comment]
@marikaj123 no definitely not :-)

**Comment by anneqm** [comment]
@kaczyns Hi, do you have what is needed to finish this work?

**Comment by kaczyns** [comment]
@anneqm I have what I need from the Tekton team.  I am waiting on the rest of the Kabanero team to update the pipelines and feel comfortable with making the switch to the new Tekton version.

**Comment by marikaj123** [comment]
@anneqm - V0.5.0 GMC is this Friday and the Kabanero operator will include it.

**Comment by kaczyns** [comment]
PR is here, waiting for review: #374

**Comment by kaczyns** [comment]
This has merged and will be included in Kabanero 0.5.0.
